From a142e2ebbc7368185a5b6605c7dd61c8728dc257 Mon Sep 17 00:00:00 2001
From: Hiroki Negishi <hiroki.negishi.bx@renesas.com>
Date: Mon, 13 Nov 2017 11:44:41 +0900
Subject: [PATCH 08/23] Revert "arm64: dts: r8a7795-es1: Add MFIS Lock device
 node"

This reverts commit 2a88d80f2122 ("arm64: dts: r8a7795-es1: Add
MFIS Lock device node")

Signed-off-by: Hiroki Negishi <hiroki.negishi.bx@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
index 2695eff..30f72bc 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
@@ -1,7 +1,7 @@
 /*
  * Device Tree Source for the r8a7795 ES1.x SoC
  *
- * Copyright (C) 2016-2017 Renesas Electronics Corp.
+ * Copyright (C) 2016 Renesas Electronics Corp.
  *
  * This file is based on the arch/arm64/boot/dts/renesas/r8a7795.dtsi
  *
@@ -877,13 +877,6 @@
 			       <&ipmmu_mp1 30>, <&ipmmu_mp1 31>;
 		};
 
-		mfis: mfis@e6260000 {
-			compatible = "renesas,mfis-lock-r8a7795",
-				     "renesas,mfis-lock";
-			reg = <0 0xe6260000 0 0x1000>;
-			#hwlock-cells = <1>;
-		};
-
 		pfc: pfc@e6060000 {
 			compatible = "renesas,pfc-r8a7795";
 			reg = <0 0xe6060000 0 0x50c>;
-- 
1.9.1

